website/content/docs/configuration/controller.mdx

```diff
@@ -14,7 +14,7 @@ controller {
   name = "example-controller"
   description = "An example controller"
   database {
-    url = "postgresql://<username>:<password>@10.0.0.1:5432/<database_name>"
+    url = "postgresql://<username>:<password>@10.0.0.1:5432/<database_name>?sslmode=disable"
```

Comment:
As it's not advised to run postgres like this in production environments, shouldn't you make another example for development environments?

Reply:
You should separate your use case from the provided example, indicating that if your database is setup in insecure mode, such as the case in development environments, that the connection string addition is needed. The default should remain the safe, best practices example.